### ISSUE TITLE
Include relationships in augmentation

### ISSUE THREAD

**Issue by edalzell** [doublethreedigital/runway#70]
### What would you like to see added?

I have `Orphans` that have many `OrphanUpdates` via the `updates()` relationship:

```php
<?php

namespace App\Models;

use App\Models\Country;
use App\Models\OrphanUpdate;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Orphan extends Model
{
    use HasFactory;

    public function country()
    {
        return $this->belongsTo(Country::class);
    }

    public function updates() {
        return $this->hasMany(OrphanUpdate::class);
    }
}
```

Would be great if I could do this:
```
{{ runway:orphan where="id:{orphan}" }}
DETAIL PAGE
<img src="{{ image }}">
<h2>{{ name }}</h2>
<p>{{ summary }}</p>

<h3>Updates</h3>
<ul>
    {{ updates }}
        <li>
            {{ text }}
        </li>
    {{ /updates }}
</ul>
{{ /runway:orphan }}
```

### Is there a workaround you can use in the meantime?

query the `orphanupdate` resource directly.

### Priority wise - where's this feature for you?

This is just a nice to have. I can deal without.

**Comment by duncanmcclean** [comment]
Ah good shout! Might have to do something funky to get it working without building a `HasMany` fieldtype (which will be a pain to build or at least I think it will be).

I'll figure something out though.

**Comment by edalzell** [comment]
@duncanmcclean no pressure at all, but is this something that will be done in the next week or so? If not, we have to walk the client through the workaround method. Again, not urgent at all, merely an expectations thing.

**Comment by duncanmcclean** [comment]
> @duncanmcclean no pressure at all, but is this something that will be done in the next week or so? If not, we have to walk the client through the workaround method. Again, not urgent at all, merely an expectations thing.

I'll see if I can make any further progress with the `HasMany` fieldtype. If I don't get that done in time, you might have to figure out some sort of workaround - which I imagine would mean overriding the Runway tag or writing your own.

**Comment by duncanmcclean** [comment]
Released the HasMany fieldtype, should hopefully solve your problem. Released in [v2.1.21](https://github.com/doublethreedigital/runway/releases/tag/v2.1.21)
